.travis.yml

```diff
@@ -39,10 +39,10 @@ install:
   - Rscript -e "tinytex::install_tinytex()"
   - echo $PATH
   - which tlmgr
-  - export PATH=${PATH}:$HOME/bin
+  - export PATH=$HOME/bin:${PATH}
```

Comment:
My current thinking is that this is what was missing before that was preventing tlmgr from installing packages on the fly. At least in [travis build #120](https://travis-ci.com/github/CoVPN/correlates_reporting/jobs/488243407), I see that it appears to now be installing tex packages on the fly. E.g., line 2155:

````
tlmgr: package repository http://ctan.math.utah.edu/ctan/tex-archive/systems/texlive/tlnet (verified)
[1/1, ??:??/??:??] install: morefloats [5k]
running mktexlsr ...
done running mktexlsr.
````

Reply:
From what I recall looking at builds yesterday, `tinytex::tlmgr_install()` was actually working fine (will try to find a reference build) for installation. From this, I’d guess the issue has something to do with whatever dependency is using that TeX package being unlinked to the correct TeX distribution.